scripts/linters/html_linter.py

```diff
@@ -33,6 +33,16 @@
 # pylint: enable=wrong-import-position
 
 
+def _htmllint_trimmer(lint_messages):
```

Comment:
Ditto move it to the class and rename this to the same name.

Reply:
Done.